todo_react/todo-react/src/components/ToDoItem.tsx

```diff
@@ -0,0 +1,52 @@
+import React, {useEffect, useRef, useState} from 'react';
+import {ITodo} from "../models";
+
+interface TodoProps {
+    todo: ITodo
+    deleteTodo: (todo: ITodo) => void
+    doneTodo: (todo: ITodo) => void
+    editTodo: (todo: ITodo, value: string) => void
+}
+
+export function ToDoItem(props: TodoProps) {
+    const [edit, setEdit] = useState(false)
```

Comment:
`edit` is a verb – used for functions (some action), we need `isEditing` – boolean value (status)

check something like https://github.com/kettanaito/naming-cheatsheet